1.12/Dockerfile.fedora

```diff
@@ -0,0 +1,40 @@
+FROM registry.fedoraproject.org/f30/s2i-base:latest
+
+ENV NAME=golang \
+    VERSION=1.12 \
+    ARCH=x86_64
+
+ENV SUMMARY="Platform for building and running Go $VERSION based applications" \
+    DESCRIPTION="Go $VERSION available as container is a base platform for \
+building and running various Go $VERSION applications and frameworks. \
+Go is an easy to learn, powerful, statically typed language in the C/C++ \
+tradition with garbage collection, concurrent programming support, and memory safety features."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Go $VERSION" \
+      io.openshift.tags="builder,golang,golang18,rh-golang18,go" \
```

Comment:
The version of golang is 1.12, But there is mentioned version 18. Is it intended?

Reply:
It appears tags haven't been updated for quite a while :/ Changes tags to golang112